integration/app/test-demo/src/containers/MeetingForm.tsx

```diff
@@ -0,0 +1,88 @@
+import {
+  Flex,
+  FormField,
+  Input,
+  Modal,
+  ModalBody,
+  ModalHeader,
+  PrimaryButton,
+  useMeetingManager,
+} from 'amazon-chime-sdk-component-library-react';
+import React, { ChangeEvent, FC, FormEvent, useState } from 'react';
```

Comment:
We don't need to import `FC` if we use `React.FC` below

Reply:
Changed all `FC` to `React.FC`.